Sources/Testing/ABI/Encoded/ABI.EncodedTest.swift

```diff
@@ -74,9 +74,13 @@ extension ABI {
     var isParameterized: Bool?
 
     /// The tags associated with the test.
-    ///
-    /// - Warning: Tags are not yet part of the JSON schema.
-    var _tags: [String]?
+    var tags: [String]?
+
+    // The bugs associated with the test.
+    var bugs: [Bug]?
+
+    /// The time limits associated with the test.
+    var timeLimit: Int?
```

Comment:
```suggestion
    var timeLimit: Double?
```

Subsecond precision is possible, at least nominally.

Reply:
ACK